contacts.js

```diff
@@ -10,6 +10,10 @@ var submitContacts = () => {
 		var html = contactItem(name, phone);
 
 		$('#contact-list').append(html);
+		$('#contact-name').val('');
+		$('#contact-phone').val('');
+
+		return false;
```

Comment:
`return false;` is not needed here because you already have `event.preventDetault()` on top